### PR TITLE
Jshint fixes

### PR DESCRIPTION
**Pull request by valdrinkoshi**

### CODE REVIEW

**Comment by valdrinkoshi** [comment]
@dbaq FYI. I've updated signup.controller to have the same style as the other controllers (injection in one line). There are some issues like camelCase that we can't guarantee since the service accepts those keys. 
@jonbwhite, is it possible to have the parameters of the services in camel case? e.g. batch services to have startTime instead of start_time

File: app/scripts/auth-interceptor.service.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ angular.module('HomeCooked.services')
           config.headers = config.headers || {};
           var credential = CacheService.getCache('hccredential');
           if (credential) {
-            config.headers.Authorization = credential.token_type + ' ' + credential.access_token;
+            config.headers.Authorization = credential['token_type'] + ' ' + credential['access_token'];
```
Comment (valdrinkoshi) [inline app/scripts/auth-interceptor.service.js]: our service returns these keys (see https://github.com/hmecookd/HomeCooked/blob/master/app/scripts/login.service.js#L16)
@jonbwhite can we have the /connect/ service return tokenType and accessToken instead? (camelCase)

**Comment by dbaq** [comment]
I see, technically you can add a comment line before the object ```/*jshint camelcase: false */ ```

but I am going to see if there is an option in jshintrc file

**Comment by dbaq** [comment]
that's interesting, the camelcase options is deprecated, we need to use the JSCS project for code style

https://github.com/johnpapa/angular-styleguide#jscs

**Comment by valdrinkoshi** [comment]
alright, let me revert the camelCase stuff i changed, I'll remove the camel case rule too

**Comment by valdrinkoshi** [comment]
what about the "too long" rule?

**Comment by dbaq** [comment]
Same thing, all the deprecated options (http://jshint.com/docs/options/) should be moved to JSCS.

**Comment by valdrinkoshi** [comment]
shall we keep these rules for now? we'll update on the go. To me they seem good enough for now

**Comment by dbaq** [comment]
yep, looks good. are you going to add jscs support in this PR?

**Comment by valdrinkoshi** [comment]
let me have a look at that, if it's too many changes i'll do in a separate PR

**Comment by valdrinkoshi** [comment]
Actually, better to tackle this in a separate PR. I'm not so sure about Jshint or jscs. Have a look at this comparisons http://www.sitepoint.com/comparison-javascript-linting-tools/

**Comment by valdrinkoshi** [comment]
FYI @dbaq Moved discussion to https://github.com/hmecookd/HomeCooked/issues/13

**Pull request merged**